tests/Fixer/Casing/ClassReferenceNameCasingFixerTest.php

```diff
@@ -197,14 +197,14 @@ public static function provideFixCases(): iterable
                 \Closure::bind(fn () => null, null, new class {});
 
                 Foo\Bar::bind(fn () => null, null, new class {});
-                \A\B\\Bar::bind(fn () => null, null, new class {});
```

Comment:
Here you can see why consistent CS is important, here the CS was not consistent even on a single line!